terra-jupyter-base/Dockerfile

```diff
@@ -109,6 +109,11 @@ RUN pip3 -V \
  # For gcloud alpha storage support.
  && pip3 install google-crc32c --target /usr/lib/google-cloud-sdk/lib/third_party
 
+RUN git clone https://github.com/broadinstitute/cromshell.git \
+ && cd cromshell \
+ && git checkout cromshell_2.0 \
+ && pip install .
```

Comment:
100% TOL – I do like the idea of test-running the tool every time to make sure that a future change in a related area (like a `pip` version upgrade) does not remove it from the path.

e.g.
https://github.com/broadinstitute/dsp-scripts/blob/master/cromwell/percona-cromwell-db-migration/Dockerfile#L18

Reply:
This makes sense. I have added version command check in dockerfile.